warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/reindex/ShardReindexJob.java

```diff
@@ -197,10 +268,61 @@ private Job setupJob() throws IOException, ParseException, AccumuloException, Ta
         FileOutputFormat.setOutputPath(j, new Path(jobConfig.outputDir));
         j.setOutputFormatClass(MultiRFileOutputFormatter.class);
 
+        // finished with the accumulo client
+        this.accumuloClient.close();
+
         return j;
     }
 
-    public static Collection<Range> buildRanges(String start, String end, int splitsPerDay) throws ParseException {
+    private void validateTablesExist(String[] tableNames) throws AccumuloException {
+        for (String table : tableNames) {
+            try {
+                Map<String,String> tableProperties = accumuloClient.tableOperations().getTableProperties(table);
+                if (tableProperties == null) {
+                    throw new IllegalArgumentException("configured output table: " + table + " does not exist");
+                }
+            } catch (TableNotFoundException tnfe) {
```

Comment:
I am okay with this pattern since we're converting a checked exception into an unchecked one. This is us saying that if the table fails, there is no reasonable recovery. One thing I would do is add the originating exception to the causedBy of the one we're throwing for stacktrace traceability.

Reply:
sounds reasonable to me.